src/components/MovieClassics.jsx

```diff
@@ -0,0 +1,80 @@
+import { Link, useNavigate} from "react-router-dom";
+import { useState, useEffect } from "react";
+import Lottie from "lottie-react";
+import animation from "../assets/animations/loading.json";
+import "./css/movielist.css";
+import arrow from "/arrow.svg";
+
+export const MovieClassics = () => {
+  const [loading, setLoading] = useState(true);
+  const [movies, setMovies] = useState([]);
+  const navigate = useNavigate();
+
+  const fetchApi = () => {
+    fetch(
+      `https://api.themoviedb.org/3/movie/top_rated?api_key=e185d0927f85272fbd0fd2526ecf0657&language=en-US&page=2`
+    )
+      .then((response) => response.json())
+      .then((movies) => {
+        setMovies(movies);
+        console.log(movies.results);
+        setLoading(false);
+      })
+      .catch((error) => {
+        console.error("Error fetching movies:", error);
+        setLoading(false);
```

Comment:
since you set Loading to false regardless of error or not, you can add a .finally() after the catch that will be executed for all the situations